Tribler/Test/Core/CreditMining/mock_creditmining.py

```diff
@@ -3,8 +3,12 @@
 
 Written by Ardhi Putra Pratama H
 """
+from mock import MagicMock
```

Comment:
Please avoid using the `mock` framework unless you absolutely need it. Introducing this framework adds another dependencies and requires updating all testing machines. Instead, you can use `MockObject`.